src/SPC/builder/windows/WindowsBuilder.php

```diff
@@ -298,6 +322,17 @@ public function sanityCheck(mixed $build_target): void
                 }
             }
         }
+
+        // sanity check for php-cgi
+        if (($build_target & BUILD_TARGET_CGI) === BUILD_TARGET_CGI) {
+            logger()->info('running cgi sanity check');
+            FileSystem::writeFile(SOURCE_PATH . '\php-cgi-test.php', '<?php echo "<h1>Hello, World!</h1>"; ?>');
```

Comment:
place this with our other tests I'd say

Reply:
This is the right place. The other tests for other OS is in `UnixBuilderBase` and win is not unix.